crates/goose/src/providers/test_helpers.rs

```diff
@@ -0,0 +1,58 @@
+#[cfg(test)]
```

Comment:
nit: helpers make sense but i'd rec we either make these general purpose if they're at top level or move them into the ollama module if not

Reply:
removed and put it in ollama module for now